doc/nrf/ug_nrf5340.rst

```diff
@@ -99,22 +107,25 @@ These samples must be programmed to the application core, in the secure domain.
 Building and programming a sample
 *********************************
 
-You must program both the network sample and one of the supported :ref:`Bluetooth Low Energy samples <ble_samples>`.
+Depending on the chosen sample, you must program just the application core (e.g. when using NFC samples) or both the network and the application cores.
+
+.. note::
+  On nRF53 the application core is responsible for starting the network core and connecting its GPIO pins. Therefore, to run any sample on nRF53, the application core must be programmed (e.g. with the :ref:`zephyr:hello_world` sample) even if the firmware is supposed to run only on the network MCU. For details see the code in: ``zephyr/boards/arm/nrf5340_dk_nrf5340/nrf5340_cpunet_reset.c`` 
 
 Build and program both samples separately by following the instructions in :ref:`gs_programming_ses`.
 Make sure to use ``nrf5340_dk_nrf5340_cpunet`` as board name when building the network sample, and ``nrf5340_dk_nrf5340_cpuapp`` when building the application sample.
 
 .. important::
-   When programming the samples from |SES|, you might get an error message stating that the target cannot be identified.
+   When programming the samples from an old version of |SES|, you might get an error message stating that the target cannot be identified.
```

Comment:
I have a SES v4.40a, which uses JLink v6.60 and has the nRF53 target definition and is able to program nRF53 (though I have issues programming the Network core). But I decided to keep this section for now, to support the users of older SES versions. I leave it up to @ru-fu and @b-gent to decide whether or not to keep it for the release.

Reply:
We need to wait for a decision on which SES version will be used for the release. If that version fully supports nRF53, we should state that it is required and remove this section. If not, we should leave the section in.

Reply:
Any news on the official SES version for NCS 1.2.0? @shantha-14?